resources/restore.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: stolon-restore
+spec:
+  template:
+    metadata:
+      name: stolon-restore
+    spec:
+      containers:
+      - name: backup
+        imagePullPolicy: Always
+        image: apiserver:5000/stolon-backup:0.0.1
+        command: ["/restore.sh"]
+        env:
+          - name: STOLON_BACKUP_DB
+            value: "{{STOLON_BACKUP_DB}}"
+          - name: STOLON_BACKUP_FILE
+            value: "{{STOLON_BACKUP_FILE}}"
+        volumeMounts:
+          - mountPath: /backups
+            name: backups
+          - mountPath: /etc/secrets/stolon
+            name: stolon
+      volumes:
+        - name: backups
+          hostPath:
+            path: /var/backups
+        - name: stolon
+          secret:
+            secretName: stolon
+      restartPolicy: OnFailure
```

Comment:
I guess if we backup something, we maybe want that it restores, so its better to periodically check that backups can be restored on some dummy DB. From my experience it not always works, sometime backups are broken and when it needed (backup always a last hope) it not work